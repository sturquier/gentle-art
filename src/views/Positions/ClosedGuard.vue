<template>
  <h1>Closed guard</h1>
</template>
