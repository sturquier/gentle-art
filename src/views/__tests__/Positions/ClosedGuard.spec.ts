import { describe, it, expect } from 'vitest'
import { mount } from '@vue/test-utils'

import ClosedGuard from '../../Positions/ClosedGuard/ClosedGuard.vue'

describe('ClosedGuard', () => {
  it('renders properly', () => {
    const wrapper = mount(ClosedGuard)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
