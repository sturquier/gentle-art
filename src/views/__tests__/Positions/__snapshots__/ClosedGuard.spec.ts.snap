// Vitest Snapshot v1

exports[`ClosedGuard > renders properly 1`] = `"<h1>Closed guard</h1>"`;
